blenderproc/python/sampler/RandomWalk.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+from typing import List
+
+def random_walk(total_length: int, dims: int, step_magnitude: float = 1.0, window_size: int = 1, 
```

Comment:
Can we rename that to `random_camera_walk`?

Reply:
A wait that works for everything?

Reply:
It's not bound to `camera_walks` it could also be used for objects.